sky/sdk/example/game/lib/sprite_box.dart

```diff
@@ -36,6 +50,15 @@ class SpriteBox extends RenderBox {
   // Transformation mode
   SpriteBoxTransformMode _transformMode;
 
+  void set transformMode (SpriteBoxTransformMode value) {
+    if (value == _transformMode)
+      return;
+    _transformMode = value;
+
+    // Invalidate stuff
+    if (attached) performLayout();
```

Comment:
Unless if (foo) return; is special, seems we should pick a one line if style and stick to it.  @Hixie likely already did.